mimic_2_data_analysis.ipynb

```diff
@@ -0,0 +1,3433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">dLine #2.    dat = pd.DataFrame(adata[:, var_names].X, columns = var_names)
</pre><p> </p><p>Not sure whether I like the name <code>dat</code> to be honest. Can we find something better? <code>data</code> is of course also not the most descriptive of all names :)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>